cmd/minishift/cmd/status.go

```diff
@@ -43,6 +44,14 @@ DiskUsage:  {{.DiskUsage}}
 CacheUsage: {{.CacheUsage}} (used by oc binary, ISO or cached images)
 `
 
+var statusFormatWithRegistration = `Minishift:  {{.MinishiftStatus}}
+Profile:    {{.ProfileName}}
+OpenShift:  {{.ClusterStatus}}
+RHSM: 	    {{.Registration}}
```

Comment:
I would place it at the bottom, so it does not affect scripts that might use line numbers